### PR TITLE
fix flow type errors

### PR DESCRIPTION
**Pull request by FredKSchott**
This gets flow passing again on the codebase (a result of running #810)

### CODE REVIEW

File: src/common/utils.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ function toRippledAmount(amount: Amount): RippledAmount {
   if (amount.currency === 'XRP') {
     return xrpToDrops(amount.value)
   }
+  // $FlowFixMe: amount.issuer is not a Amount type property. Safe to remove?
```
Comment (intelliot) [inline src/common/utils.js]: Good catch. `issuer` is actually the term that we use in rippled. We should add `issuer` to the `Amount` type, and keep using it here. (No hurry, but in a future release, we may deprecate `counterparty`.)

Comment (intelliot) [reply src/common/utils.js]: (There are other areas in the code that will need to be updated to accommodate this)

File: src/transaction/settings.js [hunk 2]
```diff
@@ -8,17 +8,36 @@ const validate = utils.common.validate
 const AccountFlagIndices = utils.common.constants.AccountFlagIndices
 const AccountFields = utils.common.constants.AccountFields
 import type {Instructions, Prepare} from './types.js'
-import type {Settings} from './settings-types.js'
+
+type Settings = {
+  passwordSpent?: boolean;
```
Comment (intelliot) [inline src/transaction/settings.js]: I believe we prefer to use commas instead of semicolons here.

File: src/common/utils.js (hunk 1 above)
Comment (FredKSchott) [reply src/common/utils.js]: Say more about the other areas that need to be changed? They'll need to accommodate by  setting `issuer`? Read from `issuer`?

Comment (intelliot) [reply src/common/utils.js]: @FredKSchott Right. For example, here are places where I think we'll need to [set issuer](https://github.com/ripple/ripple-lib/blob/develop/src/ledger/parse/amount.js#L17) or [read from issuer](https://github.com/ripple/ripple-lib/blob/develop/src/ledger/orderbook.js#L53). We can do this in a later PR though.

**Comment by intelliot** [comment]
We can clean up counterparty/issuer in a later PR :) Merging.

**Comment by FredKSchott** [comment]
👍 👍 👍

**Pull request merged**